formidable/forms/field_builder.py

```diff
@@ -97,13 +97,15 @@ def get_disabled(self):
     def get_initial(self):
         if self.field_is_dict and 'defaults' in self.field:
             defaults = self.field['defaults']
+            defaults = list(filter(None, defaults))
             if len(defaults) > 0:
                 if (self.field_class == MultipleChoiceField
                         or self.field.get('multiple', False)):
                     return defaults
                 return defaults[0]
         elif hasattr(self.field, 'defaults'):
-            defaults = self.field.defaults.all()
+            defaults = [x.value for x in self.field.defaults.all()]
+            defaults = list(filter(None, defaults))
```

Comment:
looks like this filter isn't taken into account in the tests. (I tried to run the tests without this line and there were no failure)

Reply:
and by the way, it might be a bit faster to change those two lines with:

```python
defaults = [x.value for x in self.field.defaults.all() if x.value]
```

Reply:
Done!